monitor/services/src/main/java/org/datacleaner/monitor/wizard/referencedata/AbstractReferenceDataWizardSession.java

```diff
@@ -55,23 +55,17 @@ public Integer getPageCount() {
 
     @Override
     public String finished() {
-        final DocumentBuilder documentBuilder = XmlUtils.createDocumentBuilder();
-        final TenantContext tenantContext = _wizardContext.getTenantContext();
-        final Element updatedReferenceDataSubSection = getUpdatedReferenceDataSubSection(documentBuilder);
-        final ReferenceDataDao referenceDataDao = getReferenceDataDao();
```

Comment:
DocumentBuilder, TentantContext, XmlUtils and Element are no longer used in class, so the imports can also be removed.